index.js

```diff
@@ -0,0 +1,11 @@
+// This module provides a nicer ESM module wrapper around the module generated by webpack.
+
+// NOTE: disabled eslint rules:
+// - import/no-unresolved: the lib/main.js file has likely not build yet when this runs in the CI jobs
+//   and this is also covered by automated tests (which would catch an unresolved import issue here).
```

Comment:
```suggestion
// - import/no-unresolved: in the CI jobs, the `lib/main.js` file has likely not been built yet. It's fine to
//   disable this rule because we have automated tests (which would catch an unresolved import issue
//   here).
```